cli/server.go

```diff
@@ -67,8 +67,10 @@ import (
 	"github.com/coder/coder/provisionersdk/proto"
 )
 
+type CoderServerBuilder func(*coderd.Options) *coderd.API
```

Comment:
`Coder` would be implicit here. I'd go with `ServerBuilder`. Another option that might align more with Go idioms would be `NewServerFunc`.